.github/workflows/weekly-container-build.yml

```diff
@@ -0,0 +1,24 @@
+name: Weekly Build
```

Comment:
This triggers a build on a weekly basis. May not be desired but I do it so the latest python3.X-slim base container is used so CVEs that get patched are automatically updated in the weekly build.

Again, may not be desired, discard if so.